wrapper/src/test/java/integration/refactored/container/tests/HikariTests.java

```diff
@@ -101,18 +129,179 @@ public void testOpenConnectionWithDataSourceClassName() throws SQLException {
     targetDataSourceProps.setProperty(PropertyDefinition.PLUGINS.name, "");
     // For MariaDB tests, MariaDbDataSource only accepts the url parameter.
     targetDataSourceProps.setProperty("url", ConnectionStringHelper.getUrl());
-    ds.addDataSourceProperty("targetDataSourceProperties", targetDataSourceProps);
+    dataSource.addDataSourceProperty("targetDataSourceProperties", targetDataSourceProps);
 
-    Connection conn = ds.getConnection();
+    final Connection conn = dataSource.getConnection();
 
     assertTrue(conn instanceof HikariProxyConnection);
-    HikariProxyConnection hikariConn = (HikariProxyConnection) conn;
+    final HikariProxyConnection hikariConn = (HikariProxyConnection) conn;
 
     assertTrue(hikariConn.isWrapperFor(ConnectionWrapper.class));
-    ConnectionWrapper connWrapper = (ConnectionWrapper) hikariConn.unwrap(Connection.class);
+    final ConnectionWrapper connWrapper = (ConnectionWrapper) hikariConn.unwrap(Connection.class);
     assertTrue(connWrapper.isWrapperFor(DriverHelper.getConnectionClass()));
 
     assertTrue(conn.isValid(10));
     conn.close();
   }
+
+  /**
+   * After getting successful connections from the pool, the cluster becomes unavailable.
+   */
+
+  @TestTemplate
+  @DisableOnTestDriver(TestDriver.MARIADB)
```

Comment:
Can you remind me why this needs to be disabled for the mariadb driver?

Reply:
currently hanging, there is a separate ticket to investigate the fix